README.md

```diff
@@ -854,17 +854,21 @@ environments:
     values:
       - git::https://git.company.org/helmfiles/global/azure.yaml?ref=master
       - git::https://git.company.org/helmfiles/global/us-west.yaml?ref=master
```

Comment:
I'm now wondering if this was never correct 🤔 Perhaps it should be `git::https://git.company.org/helmfiles@global/us-west.yaml?ref=master` ?

Reply:
Yes, you need to use the `@` to separate the folder from the URL.